### ISSUE TITLE
Advanced feature: listing and handling viewers' question, streamer TODO-list

### ISSUE THREAD

**Issue by JohnXLivingston** [JohnXLivingston/peertube-plugin-livechat#177]
During a live, viewers can ask many questions to the streamer. The streamer will not always answer them immediately.

It will be very usefull if the streamer (or room moderators) can flag messages as questions, so he can respond later without missing any of them.

These flagged messages can be displayed in a dedicated interface. They can be sorted, annotated, and mark as responded.

Modification to such flags will be visible for all moderators in real time (collaborative editing).

Moreover, it may be very useful if the streamer or moderators can add TODO tasks arbitrarily (same as above, but not related to a message in the room).

TL;DR: The interface is a list of tasks, with checkbox. These task can be sorted, and possibly linked to a chat message. The interface is accessible to all room moderators, and modifications are done in real time.

- [ ] Implement a TODO-list page
- [ ] Handle real-time multi-user task editing and sorting
- [ ] Grant access to this page to all Peertube user from the current instance, which are room moderators
- [ ] Add action to create tasks from messages in the chat
- [ ] stress test when there is a lot of messages and a lot of item in the todo list

Reference: NlNet-task-22
